external/netcoreapp/netcoreapp.depproj

```diff
@@ -6,10 +6,18 @@
   <PropertyGroup>
     <BinPlaceRef>true</BinPlaceRef>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
-    <NETCoreAppPackageVersion>2.0.0</NETCoreAppPackageVersion>
+    <NETCoreAppPackageVersion>2.0.7</NETCoreAppPackageVersion>
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
It's probably worthwhile to add a comment that says the OS-Group specific configuration provides the runtime files and the other provides the reference.